linode/manager
# change: [M3-8175] - Refactor KubeConfigDrawer to use CodeBlock and add Codeblock story

## Description 📝
Refactor KubeConfigDrawer component to use the CodeBlock component for UI consistency. I also moved CodeBlock to the components folder since it is pretty generic and added a story for it

## Changes  🔄
List any change relevant to the reviewer.
- Refactor `KubeConfigDrawer` to use `CodeBlock` component and minor cleanup
- Moved `CodeBlock` from `features` to `components` folder and decoupled it a bit from Linode Create
- Added `CodeBlock` story

## Preview 📷
(info has been obscured)
| Before  | After   |
| ------- | ------- |
| ![image](https://github.com/user-attachments/assets/62171f78-3edb-47ba-89ae-0dd33eaa0a25) | ![image](https://github.com/user-attachments/assets/548dfa6a-d215-4b7c-afd2-858d25262f01) |

![image](https://github.com/user-attachments/assets/e6240567-a146-4bee-9b29-6e6fc534d3f8)

## How to test 🧪

### Prerequisites
### Verification steps
(How to verify changes)
- Create a cluster pointing to dev API
- Once the cluster is provisioned, go to the cluster's details page
- Click on View under Kubeconfig
- Confirm new UI and no regressions
- Delete the cluster afterward

- Run storybook locally and confirm CodeBlock story

## As an Author I have considered 🤔

*Check all that apply*
